### PR TITLE
[apm consolidation] Add temporary redirects back to APM guide (8.8)

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Related to https://github.com/elastic/observability-docs/pull/3559

### CODE REVIEW

File: docs/en/observability/redirects.asciidoc [hunk 1]
```diff
@@ -140,3 +142,703 @@ Refer to <<synthetics-get-started-project>>.
...
+
+[role="exclude",id="api-otlp"]
+=== OpenTelemetry intake API
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/api-otlp.html[OpenTelemetry intake API].
+
+[role="exclude",id="api-jaeger"]
+=== Jaeger event intake
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/api-jaeger.html[Jaeger event intake].
```
Comment (colleenmcginnis) [inline docs/en/observability/redirects.asciidoc:699]:
```suggestion
```

File: docs/en/observability/redirects.asciidoc [hunk 2]
```diff
@@ -140,3 +142,703 @@ Refer to <<synthetics-get-started-project>>.
...
+
+[role="exclude",id="api-config"]
+=== Elastic APM agent configuration API
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/api-config.html[Elastic APM agent configuration API].
+
+[role="exclude",id="api-otlp"]
+=== OpenTelemetry intake API
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/api-otlp.html[OpenTelemetry intake API].
```
Comment (colleenmcginnis) [inline docs/en/observability/redirects.asciidoc:694]:
```suggestion
```

File: docs/en/observability/redirects.asciidoc [hunk 3]
```diff
@@ -140,3 +142,703 @@ Refer to <<synthetics-get-started-project>>.
...
+
+[role="exclude",id="release-notes-8.0"]
+=== APM version 8.0
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/release-notes-8.0.html[APM version 8.0].
+
+[role="exclude",id="known-issues"]
+=== Known issues
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/known-issues.html[Known issues].
```
Comment (colleenmcginnis) [inline docs/en/observability/redirects.asciidoc:844]:
```suggestion

```

File: docs/en/observability/redirects.asciidoc [hunk 4]
```diff
@@ -140,3 +142,703 @@ Refer to <<synthetics-get-started-project>>.
...
+
+[role="exclude",id="release-notes"]
+=== Release notes
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/release-notes.html[Release notes].
+
+[role="exclude",id="release-notes-8.9"]
+=== APM version 8.9
+{apm-guide-move-notice}
+Go to {apm-guide-ref}/release-notes-8.9.html[APM version 8.9].
```
Comment (colleenmcginnis) [inline docs/en/observability/redirects.asciidoc:794]:
```suggestion
```

**Review by bmorelli25 — APPROVED** [review]
Clicked around a bit and everything LG!

**Pull request merged**